ci.jsonnet

```diff
@@ -345,6 +321,10 @@ local part_definitions = {
       },
     },
     solaris: {
+      "$.use.build": {
```

Comment:
`"$.use.build":` should be a private field: `"$.use.build"::`

Reply:
Good catch, done.